content/elections/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "OHBM OSSIR 2021 Committee"
+subtitle: "Make a difference: join the OS SIG Committee!"
+date: 2020-06-30
+draft: false
+sidebar: false # true or false to display the sidebar
+sidebarlogo: fresh-white-alt # From (static/images/logo/)
+include_footer: true
+---
+
+# Make a difference: join the OS SIG Committee!
+
+We are excited to announce this year's nominations for the OHBM open-science Special Interest Group (OS SIG) committee for OHBM 2021!
+
+Please self-nominate yourself [HERE](https://forms.office.com/Pages/ResponsePage.aspx?id=DQSIkWdsW0yxEjajBLZtrQAAAAAAAAAAAAMAAC9pqdJUN0xITDMxUktDT0RCUjJMVlc1OTVSRTlTRS4u!)
+
+The open-science movement is one of the best things happening in neuroscience. Hackathons, open-science room, new tools and methods, reproducibility and reliability, open access and inclusivity: the OS SIG is the place where it’s all happening!
+
+If open-science excites you and you want to help push its frontiers, please consider applying for a position in the OS SIG! 
+
+[Early career researchers](https://www.ohbmtrainees.com/), BIPOC, LGBTQ+, and people from outside Europe and North America especially encouraged to apply! 
+
+All OS_SIG members have reduced or no-cost registration for OHBM 2021!
```

Comment:
```suggestion
All OS SIG members have reduced or no-cost registration for OHBM 2021!
```

Reply:
I like it! I think I'll change "members" to "officers", just to keep that delineation throughout.